.github/workflows/e2e.yaml

```diff
@@ -29,7 +29,7 @@ jobs:
       uses: Homebrew/actions/setup-homebrew@master
 
     - name: Set up Go 1.21.x
-      uses: actions/setup-go@v3
+      uses: actions/setup-go@v4
```

Comment:
Looks like that GitHub doesn't like that version either. According to https://github.com/actions/setup-go it's probably `v5` what needs to be chosen.